sdk/src/wallet/core/builder.rs

```diff
@@ -15,7 +15,7 @@ use tokio::sync::RwLock;
 use super::operations::storage::SaveLoadWallet;
 #[cfg(feature = "events")]
 use crate::wallet::events::EventEmitter;
-#[cfg(all(feature = "storage", not(feature = "rocksdb")))]
+#[cfg(all(feature = "storage", not(feature = "rocksdb"), not(feature = "jammdb")))]
```

Comment:
Should probably remove this complicated import in favor of fully qualified path below with the cfg-if